README.md

```diff
@@ -261,7 +263,9 @@ All props are optional.
 
 **tag** { String }: The HTML tag for this element. Default: `'span'`.
 
-*Any additional props (e.g. `id`, `className`, `data-whatever`) are passed directly to the HTML element.*
+**component** { Object }: A component instance to use instead of the tag.
+
```

Comment:
I think an example is necessary, along with an explanation of why you might want to do this.